src/components/searchWrapper/searchResultsItem/index.js

```diff
@@ -153,26 +163,44 @@ class SearchResultsListItem extends Component {
                 .addClass(styles.title)
         );
 
-        // Get the icon based on the loading state.
+        // Buttion icon and text.
         let icon;
-
-        if (true === this.state.loadingCreateEmbed) {
-            icon = 'spinner fa-spin';
-        } else {
-            icon = 'plus-circle';
+        let iconText;
+
+        switch(true) {
+            // Get the icon based on the loading state.
+            case (true === this.state.loadingCreateEmbed):
```

Comment:
Since you are switching above on `true` I think you can just do `case (this.state.loadingCreateEmbed):`